src/common/utility.c

```diff
@@ -367,6 +367,9 @@ dt_logo_season_t get_logo_season(void)
 #define OSD_COORDINATES_CHR_E  "E"
 #define OSD_COORDINATES_CHR_W  "W"
 
+#define OSD_ELEVATION_ASL "Above sea level"
+#define OSD_ELEVATION_BSL "Below sea level"
```

Comment:
I would make them `static const char * ... = N_("");` so they can be translated.